frontend/src/index.ejs

```diff
@@ -32,6 +32,7 @@
       href="https://fonts.googleapis.com/css2?family=Source+Code+Pro:wght@400&display=swap"
       rel="stylesheet"
     />
+    <script src="https://cdn.jsdelivr.net/pyodide/dev/full/pyodide.js"></script>
```

Comment:
Is this required since we are loading it from local assets?

Reply:
Yes, this adds the **loadPyodide** method to global object